rosidl_runtime_py/import_message.py

```diff
@@ -29,4 +29,10 @@ def import_message_from_namespaced_type(message_type: NamespacedType) -> Any:
                 DeprecationWarning)
     module = importlib.import_module(
         '.'.join(message_type.namespaces))
+
+    # Special case for action feedback
+    if message_type.name.endswith('_FeedbackMessage'):
+        action_name, name = message_type.name.split('_', 1)
```

Comment:
I wouldn't use `split` here since it makes additional assumptions about what is not in the `message_type` string. It could either use `rsplit` to avoid that assumption or cut of the already known suffix length and use the known suffix for `name`.

Reply:
Done c1e2be3